ethcore/src/block.rs

```diff
@@ -515,7 +505,7 @@ impl IsBlock for SealedBlock {
 }
 
 /// Enact the block given by block header, transactions and uncles
-fn enact(
+pub fn enact(
```

Comment:
No need to export this outside `ethcore` right?

```suggestion
pub (crate) fn enact(
```